pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -501,7 +501,6 @@ spec:
     id: readlistpod
     apiVersion: v1
     kind: Pod
-    runNamespace: {{ .Volume.runNamespace }}
```

Comment:
for jiva, openebs pods related to volume runs in app namespace. So, is it fine to remove namespace here?

Reply:
yeah you are right